client/src/main/java/io/pravega/client/stream/impl/PositionInternal.java

```diff
@@ -16,15 +16,10 @@
 import java.util.Set;
 
 /**
- * A position has two components
- * 1. ownedSegments -- segments that can be read currently. Each ownedSegment also has an offset indicating the
+ * A position has ownedSegments -- segments that can be read currently. Each ownedSegment also has an offset indicating the
```

Comment:
We need to qualify owned segment with respect to completed segments, the segments for which this reader has reached the end but not released yet. Concretely, does `ownedSegments` include completed segments?

Reply:
Yes. Added a comment.